packages/block-library/src/query/edit/inspector-controls/index.js

```diff
@@ -101,6 +100,10 @@ export default function QueryInspectorControls( props ) {
 		return onChangeDebounced.cancel;
 	}, [ querySearch, onChangeDebounced ] );
 	const showInheritControl = isControlAllowed( allowedControls, 'inherit' );
+	const inheritControlLabel = __( 'Contents' );
+	const inheritControlHelp = __(
+		"Block will display items matching the page's context, e.g. category archive or search result"
+	);
 	const showPostTypeControl =
 		! inherit && isControlAllowed( allowedControls, 'postType' );
 	const postTypeControlLabel = __( 'Content type' );
```

Comment:
Let's go ahead and change this back to "Post Type" — it's much clearer, and with the help text I suggested otherwise it's more relative. https://github.com/WordPress/gutenberg/issues/63598#issuecomment-2241374446